app/views/incomeSources/cease/CeaseIncomeSources.scala.html

```diff
@@ -0,0 +1,218 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import _root_.implicits.ImplicitDateFormatterImpl
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.incomeSourceDetails.viewmodels.CeaseIncomeSourcesViewModel
+
+@this(
+        mainTemplate: unifiedLayout,
+        implicitDateFormatter: ImplicitDateFormatterImpl,
+        h1: h1,
+        h2: h2,
+        p: p,
+        link: link,
+        hr: hr,
+        govukTable: GovukTable
+)
+
+@(sources: CeaseIncomeSourcesViewModel,
+        isAgent: Boolean,
+        backUrl: String
+)(implicit messages: Messages,
+    user: auth.MtdItUser[_])
+
+@import implicitDateFormatter.longDate
+
+@getMessage(key: String, args: String*) = @{
+    messages(s"cease-income-sources.$key", args: _*)
+}
+
+@soleTraderBusinesses = @{ sources.soleTraderBusinesses }
+@ukProperty = @{ sources.ukProperty }
+@foreignProperty = @{ sources.foreignProperty }
+@ceasedBusinesses = @{ sources.ceasedBusinesses }
+
+@ceaseSoleTraderBusinessURL = @{
+    if(isAgent) controllers.incomeSources.cease.routes.CeaseBusinessNameController.showAgent().url
+    else controllers.incomeSources.cease.routes.CeaseBusinessNameController.show().url
+}
+
+@ceaseUkPropertyURL = @{
+    if(isAgent) controllers.incomeSources.cease.routes.CeaseUKPropertyController.showAgent().url
+    else controllers.incomeSources.cease.routes.CeaseUKPropertyController.show().url
+}
+
+@ceaseForeignPropertyURL = @{
+    if(isAgent) controllers.incomeSources.cease.routes.CeaseForeignPropertyController.showAgent().url
+    else controllers.incomeSources.cease.routes.CeaseForeignPropertyController.show().url
+}
+
+@ceaseMessage = @{
+    getMessage("cease")
```

Comment:
not sure why we need this method?

Reply:
I couldnt get getMessage to work from inside triple quotes otherwise so the Welsh translation wouldn't work

Reply:
Not sure why I couldn't get it to work earlier, removed now